src/org/jetbrains/kotlin/test/helper/actions/GeneratedTestComboBoxAction.kt

```diff
@@ -264,7 +274,7 @@ class GeneratedTestComboBoxAction(val baseEditor: TextEditor) : ComboBoxAction()
                 }
                 val methodPath = File(methodPathComponents.reversed().joinToString("/"))
                     .canonicalFile.absolutePath
-                if (methodPath == truePath) {
+                if (methodPath.asPathWithoutAllExtensions == truePath.asPathWithoutAllExtensions) {
```

Comment:
It's better to extract `truePath.asPathWithoutAllExtensions` out of loop since it performs some calculations.

Reply:
True. Though it unlikely to be a performance issue

Reply:
Sure it's not a performance issue, but just a rule of good manners.

Reply:
Applied your suggestion in [c9c6ef2](https://github.com/demiurg906/test-data-helper-plugin/pull/34/commits/c9c6ef2145f539bd394df60c7bf3db07a78c85b1)